tests/cases/standalone/common/information_schema/views.sql

```diff
@@ -0,0 +1,13 @@
+--- test information_schema.views ----
+USE public;
```

Comment:
Drop view is implemented #4231, but it is not ready to merge.
So I think you can create an independent database for this test case to avoid name confusing, just like 

https://github.com/GreptimeTeam/greptimedb/blob/da1ea253ba49fdaf0d7c9e7dd2bee46ab15b8115/tests/cases/standalone/common/view/create.sql#L5

Just remember to drop it at lat and use `public` back.

Reply:
fixed